dev/com.ibm.ws.install/bnd.bnd

```diff
@@ -55,6 +55,10 @@ instrument.disabled: true
 	org.bouncycastle:bcpg-jdk18on;version=1.78.1,\
 	org.bouncycastle:bcprov-jdk18on;version=1.78.1,\
 	org.bouncycastle:bcutil-jdk18on;version=1.78.1,\
+	org.bouncycastle:bcpg-jdk18on;version=1.78,\
+	org.bouncycastle:bcprov-jdk18on;version=1.78,\
+	org.bouncycastle:bcpg-jdk18on;version=1.78.1,\
+	org.bouncycastle:bcprov-jdk18on;version=1.78.1,\
```

Comment:
```suggestion
```
This change is not needed since it was already done with the install PR.

Reply:
You didn't remove these duplicate buildpath entries as I as saying in the review comment.  Not sure why you marked it resolved.